group-community/communication.rst

```diff
@@ -0,0 +1,51 @@
+What problem are we trying to solving?
+======================================
+
+We recognise that although a lot of people know and love IRC it does have some short commings for some people, such as:
+
+* Unless you are running a persistent connection (bouncer) you lose history
+* Not everybody likes IRC
+
+The Ansible Community Working Group is looking at making it easier for people to join the Ansible Discussions
+
+
+What's the proposed solution?
+=============================
+
+We will experiment with using `Matrix.org <https://matrix.org>`_ to allow people to connect to our Freenode channels.
+
+Matrix is an ecosystem for open and interoperable communication.
+
+How can we use Matrix?
+======================
+
+1. Create a `Matrix.org account <https://matrix.org/beta/#/register>`_
```

Comment:
swap 
https://matrix.org/beta/#/register
for 
https://riot.im/app/#/register

The best place to register is https://riot.im/app/#/register - we recommend Riot as a client for those unfamiliar with Matrix, but even if the person would like to use another client, many clients do not currently implement the registration flow (and people fall back to Riot).

https://matrix.org/beta/#/register links to a legacy client (now unmaintained)  - it will probably still work, though I can't vouch for the UX.